### DISCUSSION TITLE
Consigli per termostato Wifi

### DISCUSSION THREAD

**Discussion by laspinacristian** [indomus/forum#1423] [Hardware & Firmware]
Vorrei acquistare un termostato che abbia la possibilità di funzionare in wifi anche senza l'ausilio dell'app del produttore, quindi tramite server http.
Un po' come gli shelly e i sonoff in modalità DIY.

Conoscete qualcosa da consigliare?

**Comment by KC-inDomus** [comment]
se usi Home Assistant, puoi pensare di adottare [Tado](https://indomus.it/componenti/termostato-intelligente-tado/) integrandolo localmente tramite Apple HomeKit.
diversamente, puoi utilizzare termostati ZigBee o Z-Wave, poi però tocca a te gestirti le pianificazioni. 
di termostati che offrano un'interfaccia web non ne conosco.

**Reply by laspinacristian** [reply]
Grazie per la dritta. Per quanto riguarda Tado, per poterlo integrare in Home Assistant devo prima collegarlo al loro cloud o posso bypassare completamente la loro app anche in fase di installazione iniziale?

In sintesi sto cercando di avere il mio impianto domotico totalmente indipendente dai cloud dei vari produttori. Non mi va che qualche azienda in Germania sappia che temperatura ho impostato a casa :)

**Reply by snakuzzo** [reply]
Aperta parentesi...

A volte ho l'impressione che si esageri un po' troppo con questa cosa della privacy 

...chiusa parentesi

**Reply by KC-inDomus** [reply]
devi per forza prima collegarlo al loro cloud, non foss'altro per collegare il bridge alla tua wi-fi.
però magari puoi confondergli le acque mettendo prima il termostato in frigo, chessò

**Reply by KC-inDomus** [reply]
nooo Diego, dici.
